jenkins.py

```diff
@@ -111,6 +111,10 @@ def run():
     print('Skip (https://github.com/ruslo/hunter/issues/30)')
     sys.exit(0)
 
+  if (project_dir == 'examples/double-conversion') and (toolchain == 'analyze'):
```

Comment:
Please update `.travis.yml` instead. See repository https://github.com/ingenue/hunter and example of pull request https://github.com/ingenue/hunter/pull/56/files. Pick target branch `pkg.double-conversion`.

Reply:
Please remove this one.